ui/e2e/src/config/base.playwright.config.ts

```diff
@@ -31,16 +30,9 @@ const config: PlaywrightTestConfig = {
      */
     timeout: 5000,
   },
-  /* Run tests in files in parallel */
-  fullyParallel: true,
-  /* Fail the build on CI if you accidentally left test.only in the source code. */
-  forbidOnly: !!IS_CI,
-  /* Retry on CI only */
-  retries: IS_CI ? 2 : 0,
-  /* Opt out of parallel tests on CI. */
-  workers: IS_CI ? 1 : undefined,
+
   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
-  reporter: IS_CI ? [['html']] : [['html'], ['list', { printSteps: true }]],
```

Comment:
I think these being different was a result of testing a few things out in my previous PR, but there's not a good reason for it. Keeping them the same going forward.